tests/rados/test_ec_86.py

```diff
@@ -0,0 +1,97 @@
+"""
+Module to deploy 8+6 EC pool with custom CRUSH rules for testing
+
+"""
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.crushtool_workflows import CrushToolWorkflows
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Module to deploy 8+6 EC pool with custom CRUSH rules for testing
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    crush_obj = CrushToolWorkflows(node=cephadm)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_obj)
+
+    crush_rule = config.get("crush_rule", "rule-86")
+    crush_rules = {
+        "rule-86": """id 86
+type erasure
+step set_chooseleaf_tries 5
+step set_choose_tries 100
+step take default
+step choose indep 4 type host
+step chooseleaf indep 4 type osd
```

Comment:
ensure we have choose instead of chooseleaf

Reply:
Updated!